pdns/dnsmessage.proto

```diff
@@ -126,6 +126,34 @@ message PBDNSMessage {
     required MetaValue value = 2;
   }
   repeated Meta meta = 22;                      // Arbitrary meta-data - to be used in future rather than adding new fields all the time
+
+  enum EventType {
```

Comment:
It would be useful to have an event for when an mthread is active.

This was the idea behind the `Processing` in the original issue, but perhaps a more explicit name like `MThreadActive` would be better.

Reply:
I briefly looked into this, but atm it is not easy to do. The `eventTrace` data structure is not available in the spots where the context switching happens. I think I'll opt to postpone this until a later iteration.